### PR TITLE
Create node-i18n-utilities.yml

### PR DESCRIPTION
**Pull request by nageshlop**
Created new file .travis.yml, added support for ppc64le and verified the changes with build and test.

### CODE REVIEW

File: travis-ymls/node-i18n-utilities.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# Script License      : Apache 2.0
+#
+# ----------------------------------------------------------------------------
+dist: xenial
+arch:
+   - amd64
+   - ppc64le  
+language: node_js
+node_js:
+   - "10"
```
Comment (seth-priya) [inline travis-ymls/node-i18n-utilities.yml:18]: I see that the repo is not maintained so the latest node versions might not work but any particular reason to select version 10?

Comment (nageshlop) [reply travis-ymls/node-i18n-utilities.yml:18]: @seth-priya I referred the previous travis builds and as the project is related to javascript added nodejs 10 and tested. Also verified with nodejs 15 version its working fine. Let me know how to go about this.
Build Log: https://travis-ci.com/github/nageshlop/node-i18n-utilities/builds/213742618

Comment (seth-priya) [reply travis-ymls/node-i18n-utilities.yml:18]: I believe they have moved to another CI and that is why their travis builds may not be updated? In any case, would suggest adding the latest versions as well.

**Comment by nageshlop** [comment]
@seth-priya ready for merge.

**Review by seth-priya — APPROVED** [review]

**Pull request merged**